src/DIRAC/WorkloadManagementSystem/Executor/InputData.py

```diff
@@ -379,8 +385,6 @@ def __getSiteCandidates(self, okReplicas, vo):
                     if seStatus["DiskSE"]:
                         # Sets contain only unique elements, no need to check if it's there
                         diskLFNs.add(lfn)
-                        if lfn in tapeLFNs:
-                            tapeLFNs.remove(lfn)
                     if seStatus["TapeSE"]:
                         if lfn not in diskLFNs:
```

Comment:
Does this make sense? Or should it always be added unconditionally?

Reply:
I think it makes sense...

Reply:
Why would we only add it to the tape lfns if it hasn't alread been found at a disk SE?

Reply:
ah, nice. I changed that right now.

TBH this is old (and maybe dodgy) code that worked (or kind of...) for years, I have changed it but a bit reluctantly.